src/appleseedmaya/exporters/shadingnodeexporter.cpp

```diff
@@ -469,18 +469,9 @@ void ShadingNodeExporter::exportValue(
     }
     else if (paramInfo.paramType == "float")
     {
-        if (paramInfo.mayaAttributeType == "angle")
```

Comment:
@est77 ok but the issue is, where exactly do we have any input that asks for an angle in radians or degrees? The only shader IIRC that uses an angle is a custom anisotropic shader that besides VDMs uses a rotation which is mapped from [0,1] to 2PI radians internally (so that it can be texture mapped).